opennms-base-assembly/src/main/filtered/etc/resource-types.d/ref_mib2-powerethernet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<resource-types>
+    <resourceType name="pethMainPseGroupIndex" label="Power Ethernet (MIB-2 powerEthernet)">
```

Comment:
I'd opt for keeping the resource type definition for SNMP related data collection bits in the data-collection config rather than moving them here.

The number of resource types loaded has some affect on the performance in enumerating resources, particular when using Newts, and it is more difficult to disable them when they are here.